collectors/python.d.plugin/rabbitmq/rabbitmq.chart.py

```diff
@@ -199,6 +254,21 @@ def msg_stats(self):
         stats = fetch_data(raw_data=self.stats, metrics=VHOST_MESSAGE_STATS)
         return dict(('vhost_{0}_{1}'.format(name, k), v) for k, v in stats.items())
 
+class QueueStatsBuilder:
+    def __init__(self):
+        self.stats = None
+
+    def set(self, raw_stats):
+        self.stats = raw_stats
+
+    def id(self):
+        return (self.stats['vhost'], self.stats['name'])
```

Comment:
i think `()` is not needed, `just return x, y`

Reply:
Indeed !